web/src/app/schedules/ScheduleForm.tsx

```diff
@@ -1,10 +1,26 @@
 import React from 'react'
-import p from 'prop-types'
 import { FormContainer, FormField } from '../forms'
 import { TextField, Grid } from '@mui/material'
 import { TimeZoneSelect } from '../selection'
 
-export default function ScheduleForm(props) {
+interface Value {
```

Comment:
if we merge this one first we can import value in the Create and Edit dialog PRs that are currently open
```suggestion
export interface Value {
```